### PR TITLE
Added hover styles + spacing for the panel tabs in challenge layout

### PR DESCRIPTION
**Pull request by odicho**
Final version:

https://github.com/typehero/typehero/assets/53154523/90f9ab51-13ea-4d2f-833c-5aa6534effe4

### CODE REVIEW

**Comment by bautistaaa** [comment]
lgtm but consulting with the higher power @PickleNik

File: apps/web/src/app/[locale]/challenge/[id]/left-wrapper.tsx [hunk 1]
```diff
@@ -89,12 +89,16 @@ export function LeftWrapper({ challengeId, children, expandPanel, isDesktop }: P
...
+            ? isDesktop
+              ? 'grid-rows-3 gap-2'
+              : 'grid-cols-3 gap-0.5'
+            : 'grid-cols-3 gap-0.5'
         }`}
         ref={tabsListRef}
       >
         <TabsTrigger
-          className={`rounded-md rounded-tl-xl duration-300 data-[state=active]:bg-neutral-200 dark:data-[state=active]:bg-neutral-700 ${
+          className={`rounded-md rounded-tl-xl duration-300 hover:bg-neutral-200 data-[state=active]:bg-neutral-200 dark:hover:bg-neutral-700 dark:data-[state=active]:bg-neutral-700 ${
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/challenge/[id]/left-wrapper.tsx:101]:
```suggestion
          className={`rounded-md rounded-tl-xl duration-300 hover:bg-neutral-200/50 data-[state=active]:bg-neutral-200 dark:hover:bg-neutral-700/50 dark:data-[state=active]:bg-neutral-700 ${
```

File: apps/web/src/app/[locale]/challenge/[id]/left-wrapper.tsx [hunk 2]
```diff
@@ -106,7 +110,7 @@ export function LeftWrapper({ challengeId, children, expandPanel, isDesktop }: P
           {isCollapsed && isDesktop ? <Text className="h-4 w-4" /> : 'Description'}
         </TabsTrigger>
         <TabsTrigger
-          className={`rounded-md duration-300 data-[state=active]:bg-neutral-200 dark:data-[state=active]:bg-neutral-700 ${
+          className={`rounded-md duration-300 hover:bg-neutral-200 data-[state=active]:bg-neutral-200 dark:hover:bg-neutral-700 dark:data-[state=active]:bg-neutral-700 ${
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/challenge/[id]/left-wrapper.tsx:113]:
```suggestion
          className={`rounded-md duration-300 hover:bg-neutral-200/50 data-[state=active]:bg-neutral-200 dark:hover:bg-neutral-700/50 dark:data-[state=active]:bg-neutral-700 ${
```

File: apps/web/src/app/[locale]/challenge/[id]/left-wrapper.tsx [hunk 3]
```diff
@@ -118,7 +122,7 @@ export function LeftWrapper({ challengeId, children, expandPanel, isDesktop }: P
           {isCollapsed && isDesktop ? <FlaskConical className="h-4 w-4" /> : 'Solutions'}
         </TabsTrigger>
         <TabsTrigger
-          className={`rounded-md rounded-tr-lg duration-300 data-[state=active]:bg-neutral-200 dark:data-[state=active]:bg-neutral-700 ${
+          className={`rounded-md rounded-tr-lg duration-300 hover:bg-neutral-200 data-[state=active]:bg-neutral-200 dark:hover:bg-neutral-700 dark:data-[state=active]:bg-neutral-700 ${
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/challenge/[id]/left-wrapper.tsx:125]:
```suggestion
          className={`rounded-md rounded-tr-lg duration-300 hover:bg-neutral-200/50 data-[state=active]:bg-neutral-200 dark:hover:bg-neutral-700/50 dark:data-[state=active]:bg-neutral-700 ${
```

**Review by PickleNik — COMMENTED** [review]
i think it would look better if hover bg color differed from the active one

**Comment by odicho** [comment]
> i think it would look better if hover bg color differed from the active one

Agreed, it looks much better with the suggestions you provided! Thanks @PickleNik

**Review by PickleNik — APPROVED** [review]

**Pull request merged**